### PR TITLE
make example more common, drop box param

### LINKED ISSUE #780: Rename Box to something less spatial
Thinking forwards to annotations of time based media, it would be good to rename box to be less spatial (e.g. segment) to allow for searching within the time stream of audio.

### PR DESCRIPTION
**Pull request by azaroth42**
* Makes example more common formulation, as there were two examples in use anyway so no need to have "birds" repeated.
* Ups to 0.9.3, with new "Lost Summer" name
* Removes box parameter, closes #780 (uses user as ignored param example)

### CODE REVIEW

File: source/api/search/0.9/index.md [hunk 1]
```diff
@@ -484,17 +483,17 @@ In cases like this there are more annotations than hits as two or more annotatio
...
       "http://iiif.io/api/search/{{ page.major }}/context.json"
   ],
-  "@id":"http://example.org/service/manifest/search?q=bush+are",
+  "@id":"http://example.org/service/manifest/search?q=hand+is",
   "@type":"sc:AnnotationList",
 
   "resources": [
     {
-      "@id": "http://example.org/identifier/annotation/anno-bush",
+      "@id": "http://example.org/identifier/annotation/anno-bbird",
```
Comment (jpstroop) [inline source/api/search/0.9/index.md]: Doesn't really matter, but do you mean 'bbird" or just "bird"?

Comment (azaroth42) [inline source/api/search/0.9/index.md]: Unintentional, fixed.

**Comment by mikeapp** [comment]
:+1:

**Comment by zimeon** [comment]
:+1: (rebased to make CI happy)

**Comment by jpstroop** [comment]
:+1:

**Pull request merged**